src/Main.java

```diff
@@ -2,7 +2,9 @@
 public class Main {
     public static void main(String[] args) {
         Animal a1 = new Dog();
+        Animal a2 = new Fox();
```

Comment:
Please use a descriptive name for the animal, something like `Animal fox = new Fox();`

Reply:
Hello!, i've ready make the changes